README.md

```diff
@@ -145,6 +148,19 @@ Messenger.updateUserCredentials(username: String, email: String)
 Messenger.clearUserCredentials()
 ```
 
+**Verified**
```

Comment:
Now when I checked all that has left through official channels like this one: https://support.dixa.help/en/articles/1659-authentication-in-dixa-messenger-jwt-jwe-setup-guide-technical-documentation
I see verified being there. In code everywhere we have `verified` - therefore it might be better to leave it verified everywhere.
Which means that my suggestions to this PR https://github.com/dixahq/messenger-ios-widget/pull/242 better be reverted.
Sorry for the double effort. :(

If we would have to change verified to authenticated I think it should be done as a separate planned activity.